aws/compose.yml

```diff
@@ -15,7 +15,7 @@ services:
     container_name: orangehrm
 
   mariadb1011:
-    image: mariadb:10.11.4
+    image: mariadb:10.11.6
```

Comment:
do we need the patch version?

Reply:
I decided to keep in the event that another subversion breaks the installation